### PR TITLE
add search window

### LINKED ISSUE #6: Features
Here we can discuss new plans.
Current plans:
- add search window
- smth else?

### PR DESCRIPTION
**Pull request by alexeyhorkin**
i added search input)

### CODE REVIEW

**Comment by alexeyhorkin** [comment]
@dkurt please review

**Comment by dkurt** [comment]
Hi! It's fine for this PR but you should create separate branches for pull requests. Otherwise, after merge you will break local master branch (different commit hash comparing to remote).

File: index.html [hunk 1]
```diff
@@ -149,7 +153,7 @@
       <form action="https://github.com/apps/doska-helper/installations/new">
         <input type="submit" value="Add private repository" />
       </form>
-
+      <input id="search_repos" placeholder='Write project name'>
```
Comment (dkurt) [inline index.html:172]: Please check cursor. I have "pointer" (hand) but expected "text"

see https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

File: style.css [hunk 2]
```diff
@@ -88,3 +88,11 @@ input {
     margin-right: 10px;
     background: green;
 }
+
+#search_repos{
+  height: 25px;
+  width: 90%;
```
Comment (dkurt) [inline style.css:94]: Do we really need these sizes?

File: style.css [hunk 3]
```diff
@@ -88,3 +88,11 @@ input {
...
+  border-radius: 10px;
+  margin-bottom: 10px;
+  box-shadow: 3px 7px 11px -7px black;
+}
```
Comment (dkurt) [inline style.css]: Please add an empty line at the end of file. You may configure your text editor to do it automatically.

File: style.css (hunk 2 above)
Comment (dkurt) [reply style.css:94]: Actually, only `width`. I'd like to recommend to increase `height` (i.e. 30px)

File: index.html [hunk 4]
```diff
@@ -2,7 +2,11 @@
 
 <head>
   <!-- jQuery -->
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script
+  src="https://code.jquery.com/jquery-3.5.1.min.js"
+  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
```
Comment (dkurt) [inline index.html]: What is this sum?

File: index.html (hunk 1 above)
Comment (dkurt) [inline index.html:172]: Can you try to create similar window for admin table as well for users (`tasks_table`, panel `<div id="tasks">`)?

File: style.css [hunk 5]
```diff
@@ -88,3 +88,11 @@ input {
     margin-right: 10px;
     background: green;
 }
+
+#search_repos{
+  height: 25px;
```
Comment (dkurt) [inline style.css]: Do you mind to use 30px height?

File: index.html (hunk 4 above)
Comment (alexeyhorkin) [reply index.html]: This all about to add JQuery 3.x version instead of current 1.12.4) not such necessary, but may be helpful) anyway, as u wish)

Comment (dkurt) [reply index.html]: I'm good with newer version but why not just
```html
<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```
?

File: index.html [hunk 6]
```diff
@@ -169,5 +169,15 @@ <h1 id="project_title"></h1>
 
   </div>
 </body>
-
+<script>
+document.getElementById("search_repos").addEventListener('input', function(e){
+    
+    let tmp = $("#projects_table > tbody > tr")
```
Comment (dkurt) [inline index.html]: Why `tmp`? Please choose more meaningful name.

File: index.html (hunk 1 above)
Comment (alexeyhorkin) [reply index.html:172]: @dkurt , i think i did it... can u review?

File: index.html [hunk 7]
```diff
@@ -14,6 +14,14 @@
       window.location.replace('http://localhost:8000?code=' + searchParams.get('code'));
     }
 
+    function blind_elems(row_list, text_value){
```
Comment (dkurt) [inline index.html]: +1 whitespace
```js
function blind_elems(row_list, text_value) {
```

File: index.html [hunk 8]
```diff
@@ -35,7 +43,19 @@
       if (view.localeCompare("admin") == 0) {
         // Admin view
         let header = table.insertRow();
-        header.insertCell().innerHTML = 'User name';
+
+        // input node
+        const search_users_field = document.createElement("input");
+        search_users_field.setAttribute("id","search_user");
+        search_users_field.setAttribute("placeholder","User name");
```
Comment (dkurt) [inline index.html]: +1 whitespace
```js
        search_users_field.setAttribute("id", "search_user");
        search_users_field.setAttribute("placeholder", "User name");
```

File: index.html (hunk 1 above)
Comment (dkurt) [reply index.html:172]: Seems like the search now is case-sensitive, can you please make it insensitive?

**Review by dkurt — APPROVED** [review]
:+1: Well done! Keep it going!

**Pull request merged**